packages/central-server/app/admin/patientMerge/mergePatient.js

```diff
@@ -213,6 +213,62 @@ export async function mergePatientDeathData(models, keepPatientId, unwantedPatie
   return results;
 }
 
+export async function mergePatientProgramRegistrations(models, keepPatientId, unwantedPatientId) {
+  const existingUnwantedRegistrations = await models.PatientProgramRegistration.findAll({
+    where: { patientId: unwantedPatientId },
+  });
+
+  if (!existingUnwantedRegistrations.length) {
+    return [];
+  }
+
+  const results = [];
+
+  for (const unwantedRegistration of existingUnwantedRegistrations) {
+    // Move to keep patient and reset isMostRecent to false
+    await unwantedRegistration.update({
+      patientId: keepPatientId,
+      isMostRecent: false,
+    });
+
+    // Soft delete the registration
+    await unwantedRegistration.destroy();
+    results.push(unwantedRegistration);
```

Comment:
Had a conversation with @MeganLane29 about this. What we decided was:
- Move the registrations to the kept patient
- Soft delete the registrations (so that the registrations from the merged patient won't appear in the history of the kept patient)

Reply:
Triple headed attack!